File: pset1/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    int height;
    do{
        printf("Input a height: ");
        height = get_int();
    } while(height < 0 || height > 23);

    for(int i = 1; i<=height; i++)
        {
            for(int j = 0; j<height-i; j++) /*spaces*/
            {
                printf(" ");
            }
            for(int k = 0; k<i; k++) /*hashes*/
            {
                printf("#");
            }


            printf("  ");

            for(int k = 0; k<i; k++) /*hashes*/
            {
                printf("#");
            }

            printf("\n");
        }
}

File: pset6/mario/mario.py
import cs50
while True:
    num = input("Input a height: ")
    if int(num) > 0 and int(num) < 23:
        break

for i in range(int(num)):
    print(" " * (int(num) - i - 1),end="")
    print("#" * (i + 1),end="")
    print("  ",end="")
    print("#" * (i + 1))
